about.html

```diff
@@ -0,0 +1,175 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Lato:wght@400;700;900&display=swap"
+      rel="stylesheet"
+    />
+    <script
+      src="https://kit.fontawesome.com/162dfb3961.js"
+      crossorigin="anonymous"
+    ></script>
+    <link rel="stylesheet" href="css/main.css" />
+    <link rel="stylesheet" href="css/desktop.css" />
+    <title>Capstone Project1</title>
+  </head>
+  <body>
```

Comment:
- [ ] Kindly ensure that your about-page design matches the design guidelines given. Please take note of the background, typography, color, padding/margins 👍🏽 

| Design Guideline  | Your Design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/63473584/190246875-2f4ef15c-285b-4006-b7f8-ad38e40fc873.png)  | ![image](https://user-images.githubusercontent.com/63473584/190243267-5cff979d-9ef7-4442-a9c2-7ba946ba1d6c.png)  |
| ![image](https://user-images.githubusercontent.com/63473584/190247006-fd93c700-e53a-44aa-ab73-1a4e3f244f4d.png)  | ![image](https://user-images.githubusercontent.com/63473584/190243401-54c1fb6a-26ab-47de-ada4-33ae2fa0f1f8.png)  |